modules/rabbitmq.nix

```diff
@@ -264,6 +264,13 @@ with k8s;
           name = module.name;
         }];
       };
+
+      kubernetes.resources.podDisruptionBudgets.rabbitmq = {
+        metadata.name = module.name;
+        metadata.labels.app = module.name;
+        spec.maxUnavailable = 1;
```

Comment:
Should the `maxUnavailable` calculated based on the number of replicas etc.?

Reply:
Kubernetes supports to specify in percents, but idk how they do the rounding, so i rahter have conrete number, when you are doing eviciton you want to have all nodes running, especially in small clusters we have.